package.json

```
@@ -31,15 +31,15 @@
     "extends @decidim/browserslist-config"
   ],
   "devDependencies": {
-    "@babel/eslint-parser": "^7.23.10",
+    "@babel/eslint-parser": "^7.24.6",
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/plugin-proposal-private-methods": "^7.18.6",
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
-    "@babel/plugin-transform-classes": "^7.23.8",
-    "@babel/plugin-transform-regenerator": "^7.23.3",
-    "@babel/plugin-transform-runtime": "^7.23.9",
-    "@babel/preset-env": "^7.23.9",
-    "@babel/preset-react": "^7.23.3",
+    "@babel/plugin-transform-classes": "^7.24.6",
+    "@babel/plugin-transform-regenerator": "^7.24.6",
+    "@babel/plugin-transform-runtime": "^7.24.6",
+    "@babel/preset-env": "^7.24.6",
+    "@babel/preset-react": "^7.24.6",
     "@decidim/dev": "^0.27.4",
     "@decidim/eslint-config": "^0.27.4",
     "@decidim/stylelint-config": "^0.27.4",
```

Change the version of these packages too:

```suggestion
    "@decidim/dev": "^0.27.6",
    "@decidim/eslint-config": "^0.27.6",
    "@decidim/stylelint-config": "^0.27.6",
```